docs/ppp/pitch-23112022.md

```diff
@@ -0,0 +1,91 @@
+
+- klochowicz, aka Mariusz
+
+  - [https://github.com/klochowicz](https://github.com/klochowicz)
+  - [https://makers.bolt.fun/profile/461/mariusz](https://makers.bolt.fun/profile/461/mariusz)
+
+- luckysori, aka Lucas
+
+  - [https://github.com/luckysori](https://github.com/luckysori)
+  - [https://makers.bolt.fun/profile/534/luckysori](https://makers.bolt.fun/profile/534/luckysori)
+
+## The next steps
+
+![](https://imagedelivery.net/wyrwp3c-j0gDDUWgnE7lig/166aa91f-7136-4f4e-e35e-d4b1a7eefc00/public)
+
+1. Evaluate
+
+   1. With the outcome of the tournament we want to collect feedback from users to learn how to build the right product
+
+2. Build MVP
+
+   1. Once collected, it's all about building the product right. We aim for a ready-to-use MVP in Q1/23
+
+3. Launch public beta
+
+   1. Once stable, we will launch a public beta. This is aimed around Q3/23
+
+4. Scale
+
+   1. Once out in the public, we will launch and focus on scaling and adding new features
+
+## The links
+
+GitHub Project
+
+- [https://github.com/itchysats/10101/](https://github.com/itchysats/10101/)
+
+PPPs
+
+- Week 1: [https://makers.bolt.fun/story/ppps-week-1-tentenone--218](https://makers.bolt.fun/story/ppps-week-1-tentenone--218)
+- Week 2: [https://makers.bolt.fun/story/ppps-week-2-tentenone--264](https://makers.bolt.fun/story/ppps-week-2-tentenone--264)
+- Week 3: [https://makers.bolt.fun/story/ppps-week-3-10101--307](https://makers.bolt.fun/story/ppps-week-3-10101--307)
+- Week 4: [https://makers.bolt.fun/story/ppps-week-4-10101--351](https://makers.bolt.fun/story/ppps-week-4-10101--351)
+- Week 5: [https://makers.bolt.fun/story/ppps-week-5-10101--397](https://makers.bolt.fun/story/ppps-week-5-10101--397)
+- Week 6: [https://makers.bolt.fun/story/ppps-week-6-10101--426](https://makers.bolt.fun/story/ppps-week-6-10101--426)
+
+Other posts
+
+- Your lightning powered journey to non-custodial CFD trading [https://makers.bolt.fun/story/your-lightning-powered-journey-to-non-custodial-cfd-trading--268](https://makers.bolt.fun/story/your-lightning-powered-journey-to-non-custodial-cfd-trading--268)
+- How 10101 is bringing CFDs to Lightning [https://makers.bolt.fun/story/how-10101-is-bringing-cfds-to-lightning--352](https://makers.bolt.fun/story/how-10101-is-bringing-cfds-to-lightning--352)
+- Milestone 0.1.0 First minor release of the 10101 Wallet [https://makers.bolt.fun/story/milestone-010-first-minor-release-of-the-10101-wallet--355](https://makers.bolt.fun/story/milestone-010-first-minor-release-of-the-10101-wallet--355)
+
+Public chat
+
+- [https://matrix.to/#/%23tentenone%3Amatrix.org](https://matrix.to/#/%23tentenone%3Amatrix.org)
+
+### The vision
+
+- Sex is great, but have you tried noncustodial finance? [https://makers.bolt.fun/story/sex-is-great-but-have-you-tried-noncustodial-finance--430](https://makers.bolt.fun/story/sex-is-great-but-have-you-tried-noncustodial-finance--430)
+
+## The pitch video
```

Comment:
That's the meat of this post. I would move that up! right after the problems and solutions section (written pitch)